provision.sh

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env bash
+set -e
+
+function wrong_args() {
+  echo >&2 "Wrong number of arguments! Usage:"
+  echo >&2 "  $(basename "$0")"
+  echo >&2 "  $(basename "$0") region-name country-code-list [osm-file-download-url]"
+  echo >&2 "Example:"
+  echo >&2 "  $(basename "$0") belgium BE http://download.geofabrik.de/europe/belgium-latest.osm.pbf"
+  exit 1
+}
+
+function usage() {
+  echo "Usage:"
+  echo "  $(basename "$0")"
+  echo "  $(basename "$0") region-name country-code-list [osm-file-download-url]"
+  echo
+  echo "Example 1."
+  echo "  $(basename "$0")"
+  echo
+  echo "  Sets up backend to run in air distances routing mode."
+  echo "  WARNING: Air mode doesn't give reliable values. \
+It's only useful for evaluation, debugging, or incremental setup purpose."
+  echo
+  echo
+  echo "Example 2."
+  echo "  $(basename "$0") belgium-latest.osm.pbf BE"
+  echo
+  echo "  Use GraphHopper routing mode and upload OSM file or GraphHopper data manually. \
+The backend will be initially configured to air routing mode. When it starts for the first time, \
+use it to upload Belgium OSM file or GraphHopper data to the claimed Persistent Volume. For example:"
+  echo
+  echo "      oc rsync ~/.vrp/openstreetmap/belgium-latest.osm.pbf backend-1-7k5pl:/deployments/local/openstreetmap/"
```

Comment:
oc rsync ./belgium-latest.osm.pbf backend-1-cjtjf:/deployments/local/openstreetmap/
gives me:
error: path ./belgium-latest.osm.pbf must point to a directory

Moreover, the "backend-1-7k5pl" is a name of a running pod. If we advise users to upload the file like that, we should tell them how to find the pod's name.